src/main/resources/static/swagger.yaml

```diff
@@ -272,6 +272,27 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/PatientHealthDataResponse'
+  /v1/cm/patient/health-information/status/:
+    post:
+      tags:
+        - cm app
+      summary: Get health information status.
+      description: This API can be used to checkt the status of health information against a set of data request ids when CM app request for data.
+      parameters:
+        - $ref: '#/components/parameters/authorization'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PatientHealthDataFetchRequest'
```

Comment:
the request body of PatientHealthDataFetchRequest is already there?

Reply:
yeah, It is same as the other. If you want we can separate it out

Reply:
No, That's ok!